storm-client/src/storm.thrift

```diff
@@ -759,6 +759,7 @@ service Nimbus {
   void createStateInZookeeper(1: string key); // creates state in zookeeper when blob is uploaded through command line
 
   // need to add functions for asking about status of storms, what nodes they're running on, looking at task logs
+  set<string> getBlacklistedSupervisorIds()
```

Comment:
We don't need to expose this function to client.  `getSupervisorSummary` and `getSupervisorPageInfo` are the two places that needs this information. They are all inside nimbus.java. We can just have a private `getBlacklistedSupervisorIds` function in `nimbus.java` .

Reply:
Yes you are right. Addressed.